todo-app/src/components/CreateTodoList.js

```diff
@@ -1,3 +1,40 @@
-const CreateTodoList = () => {};
+import { useState, useRef } from "react";
```

Comment:
I would prefer if you used state and controlled components instead of ref.
See more here: https://reactjs.org/docs/uncontrolled-components.html